packages/flutter/lib/src/cassowary/term.dart

```diff
@@ -6,11 +6,16 @@ import 'equation_member.dart';
 import 'expression.dart';
 import 'param.dart';
 
+/// Represents a single term in an expression. This term contains a single
+/// indeterminate and has degree 1.
 class Term extends EquationMember {
+  /// Create term with the given variable and coefficient.
   Term(this.variable, this.coefficient);
 
+  /// The variable portion of this term.
   final Variable variable;
 
+  /// The coefficient of this term.
```

Comment:
I looked at that one first before because I noticed that when people write docs, they usually get tired when they get to the end, so the last ones need more work. :-) I've gone through the whole thing now.